src/View/mainpage/DrawROIWindow/DrawBoundingBox.py

```diff
@@ -0,0 +1,66 @@
+from PySide6 import QtWidgets, QtGui, QtCore
+from PySide6.QtWidgets import QGraphicsPixmapItem
+
+
+class DrawBoundingBox(QtWidgets.QGraphicsScene):
+    """
+    Object responsible for updating and displaying the bounds of the ROI draw.
+    """
+
+    def __init__(self, image_to_paint, pixmap_data):
+        super().__init__()
+        self.addItem(QGraphicsPixmapItem(image_to_paint))
+        self.img = image_to_paint
+        self.pixmap_data = pixmap_data
+        self.drawing = False
+        self.start_x = None
+        self.start_y = None
+        self.end_x = None
+        self.end_y = None
+        self.box = None
+
+    def draw_bounding_box(self, new_box=False):
+        if new_box:
+            self.box = QtWidgets.QGraphicsRectItem(self.start_x, self.start_y, 0, 0)
```

Comment:
```suggestion
            self.box = QtWidgets.QGraphicsRectItem(self.start_x, self.start_y,
                                                   0, 0)
```

PEP 8 compliance. Also, all functions in this file need docstrings.